generate_homepage_from_content.py

```diff
@@ -27,7 +29,8 @@
 # HTML templates
 CONTENT_CONTAINER = '<div class="content %s">%s</div>'
 WEBSITE_CONTENT_TEMPLATE = ('<span class="website-content">%s</span>\n'
-                            '<span class="website-url">%s</span>')
+                            '<span class="website-url">'
+                            '<a href="%s">%s</a></span><br><br>')
```

Comment:
I'll take out the <br><br>s and wait for Jill's change. And actually, I see she linkifies the url as well, so I'll remove that from mine to make a cleaner merge.